test/integration_test.py

```diff
@@ -34,9 +34,9 @@
 q_learning_systems = ["q_learning.anakin.rec_iql"]
 sac_systems = ["sac.anakin.ff_isac", "sac.anakin.ff_masac"]
 
-discrete_envs = ["gigastep", "lbf", "matrax", "rware", "smax"]
+discrete_envs = ["gigastep", "lbf", "matrax", "rware", "smax", "mpe"]
 cnn_envs = ["cleaner", "connector"]
-continuous_envs = ["mabrax"]
+continuous_envs = ["mabrax", "mpe"]  # todo: test mpe only in cont or disc envs?
```

Comment:
I think test only cont and make the default in the config cont as this is how we use it in benchmarks, then you should be able to remove all the if `env == "mpe"` from this file :smile: